src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -80,7 +80,7 @@ public static function parse(string $content, $schemaOrCallable = null)
                 } catch (\Exception $e) {
                     $valid = false;
                 }
-            } elseif (!\is_array($schemaOrCallable) && is_file((string) $schemaOrCallable)) {
+            } elseif (is_file($schemaOrCallable)) {
```

Comment:
we can also improve the error message

Reply:
Any suggestions? imho, the error message is still fine.

Reply:
Maybe just: `Invalid XSD file: "%s".`